applications/crossbar/doc/internationalization/numbers.md

```diff
@@ -83,33 +83,33 @@ This is a set of regexes to group numbers by type and are not used for routing.
 
     "classifiers":{
         "tollfree_us":{
-            "regex":"^\\+1(800|888|877|866|855)\\d{7}$",
+            "regex":"^\\+1((800|888|877|866|855)\\d{7})$",
```

Comment:
You're right. I thought R15B03 didn't support regexp comments, but indeed it does.
And sorry for the response time, I didn't see your comment earlier.